tap_jira/__init__.py

```diff
@@ -109,15 +114,18 @@ def sync():
 def main():
     args = get_args()
 
+    jira_config = args.config
+    # jira client instance
+    jira_client = Client(jira_config)
+
```

Comment:
Why do we need to have a separate variable called jira_config? args.config could be used to get jira_client right?

Reply:
Yes, args.config can be used, we've just assigned it to a variable for reuse later.